test/test-ghe-backup.sh

```diff
@@ -463,9 +463,12 @@ begin_test "ghe-backup stores version when not run from a clone"
   # Make sure this doesn't exist
   rm -f "$GHE_REMOTE_DATA_USER_DIR/common/backup-utils-version"
 
-  mv "$ROOTDIR/.git" "$ROOTDIR/.gittmp"
-  ghe-backup
-  mv "$ROOTDIR/.gittmp" "$ROOTDIR/.git"
+  tmpdir=$TRASHDIR/ghe-backup-stores-version-$RANDOM
```

Comment:
@sbryant indeed, but I want it to be inside `TRASHDIR`, so it's automatically garbage collected. We could use additional args but macOS compatibility will make it harder. The randomly generated name should be totally safe and doesn't have compat issues.